haiku-apps/tesseracttranslator/tesseracttranslator-0.1.recipe

```diff
@@ -0,0 +1,44 @@
+SUMMARY="Haiku OCR translator"
+DESCRIPTION="An image to text translator add-on using Tesseract OCR engine."
+HOMEPAGE="https://github.com/threedeyes/TesseractTranslator"
+COPYRIGHT="2013 3dEyes"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="http://github.com/threedeyes/TesseractTranslator/archive/master.zip"
+CHECKSUM_SHA256="c6bbdad4cdde74eb4c262ead281eadf563960aac8dc4f58fb30dfb70d5d16fd9"
+SOURCE_DIR="TesseractTranslator-master"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+
+PROVIDES="
+	tesseracttranslator$secondaryArchSuffix = $portVersion
+	addon:TesseractTranslator$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libtesseract$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libtesseract$secondaryArchSuffix
+	devel:liblept$secondaryArchSuffix
+	"
+#	tesseract${secondaryArchSuffix}_devel
+#	leptonica${secondaryArchSuffix}_devel
+
+BUILD_PREREQUIRES="
+	cmd:make
+	cmd:gcc
```

Comment:
This needs to have a $secondaryArchSuffix, and there also needs to be a `cmd:ld$secondaryArchSuffix`.